game/graphics/pipelines/opengl.cpp

```diff
@@ -233,200 +148,163 @@ static std::shared_ptr<GfxDisplay> gl_make_display(int width,
   g_gfx_data->imgui_filename = file_util::get_file_path({"imgui.ini"});
   g_gfx_data->imgui_log_filename = file_util::get_file_path({"imgui_log.txt"});
   ImGuiIO& io = ImGui::GetIO();
+  io.ConfigFlags |= ImGuiConfigFlags_NoMouseCursorChange;  // We manage the mouse cursor!
+  if (!Gfx::g_debug_settings.monospaced_font) {
+    std::string font_path =
+        (file_util::get_jak_project_dir() / "game" / "assets" / "fonts" / "Roboto-Medium.ttf")
+            .string();
+    if (file_util::file_exists(font_path)) {
+      io.Fonts->AddFontFromFileTTF(font_path.c_str(), Gfx::g_debug_settings.imgui_font_size);
```

Comment:
Can you add the following before this line?
```cpp

    static const ImWchar ranges[] = {
        0x0020, 0x00FF,  // Basic Latin + Latin Supplement
        0x2000, 0x206F,  // General Punctuation
        0x3000, 0x30FF,  // CJK Symbols and Punctuations, Hiragana, Katakana
        0x3131, 0x3163,  // Korean alphabets
        0x31F0, 0x31FF,  // Katakana Phonetic Extensions
        0xAC00, 0xD7A3,  // Korean characters
        0xFF00, 0xFFEF,  // Half-width characters
        0xFFFD, 0xFFFD,  // Invalid
        0x4e00, 0x9FAF,  // CJK Ideograms
        0,
    };
```
This will make it so more than just the latin alphanumeric characters get loaded.

Reply:
Surely `ranges` has to be used somewhere, right?

Also `Wchar` makes me nervous -- is this a windows only thing / what happens on linux?

Reply:
Oops yeah I forgot to tell you where it's used. You pass it to the `AddFontFromFileTTF` call, it's the fourth argument I believe.

Reply:
As for the type... these are just unicode code points, so they normally cap out at `0xFFFF`. I don't even know if there's a way to get imgui to render characters above that code point.